arcgis-ios-sdk-samples/Display information/Sketch on the map/Sketch.storyboard

```diff
@@ -20,38 +21,34 @@
-                                            <rect key="frame" x="20" y="11" width="204" height="32"/>
-                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                                            <segments>
-                                                <segment title="" image="Point"/>
-                                                <segment title="" image="Polyline"/>
-                                                <segment title="" image="FreehandPolyline"/>
-                                                <segment title="" image="Polygon"/>
-                                                <segment title="" image="FreehandPolygon"/>
-                                            </segments>
-                                            <connections>
-                                                <action selector="geometryValueChanged:" destination="DMe-yp-yH3" eventType="valueChanged" id="UPW-qg-uR6"/>
-                                            </connections>
-                                        </segmentedControl>
-                                    </barButtonItem>
-                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="heC-81-cSK"/>
-                                    <barButtonItem image="Undo" id="Xfw-dX-zVQ">
+                                    <barButtonItem systemItem="undo" id="Xfw-dX-zVQ">
                                         <connections>
                                             <action selector="undo" destination="DMe-yp-yH3" id="hpK-dm-9G0"/>
                                         </connections>
                                     </barButtonItem>
-                                    <barButtonItem image="Redo" id="fQO-v5-LKM">
+                                    <barButtonItem systemItem="redo" id="fQO-v5-LKM">
                                         <connections>
                                             <action selector="redo" destination="DMe-yp-yH3" id="c6W-St-RTy"/>
                                         </connections>
                                     </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="32P-6u-mlV"/>
+                                    <barButtonItem systemItem="add" id="h9e-Af-91p">
+                                        <connections>
+                                            <action selector="addGeometryButtonTapped:" destination="DMe-yp-yH3" id="kRO-VT-uVd"/>
+                                        </connections>
+                                    </barButtonItem>
                                     <barButtonItem style="plain" systemItem="flexibleSpace" id="L2j-qC-Sng"/>
-                                    <barButtonItem image="Discard" id="szF-EX-1S3">
+                                    <barButtonItem image="trash" catalog="system" id="szF-EX-1S3">
```

Comment:
Can you try to choose "trash" from the "System Item" drop down list in storyboard? Not sure why storyboard decides to add it like this. 🤔